### PR TITLE
Change "run ML" phrases to more appropriate ones in vignettes

### LINKED ISSUE #253: Use the phrase "model training" rather than "ML" where appropriate
From https://github.com/openjournals/joss-reviews/issues/3073#issuecomment-828720647:

> In a number of places across the vignette, "machine learning" (e.g. "run machine learning" in the "Preprocessing data" vignette) is used in place of a likely more appropriate "model training"/"train model"

### PR DESCRIPTION
**Pull request by BTopcuoglu**
## Issues
- Resolves #253.

## Change(s) made
- Edited the vignettes to change "run ML" phrases to more appropriate ones. 

## Checklist

(~Strikethrough~ any points that are not applicable.)

- ~[ ] Write unit tests for any new functionality.~
- ~[ ] Update roxygen comments & vignettes if there are any API changes.~
- ~[ ] Update `NEWS.md` if this includes any user-facing changes.~
- [x] The check workflow succeeds on your most recent commit.

### CODE REVIEW

File: vignettes/introduction.Rmd [hunk 1]
```diff
@@ -18,7 +18,7 @@ knitr::opts_chunk$set(
 
 The goal of `mikropml` is to make supervised machine learning (ML) easy for you
 to run while implementing good practices for machine learning pipelines. All you
-need to run ML is one function: `run_ml()`. We've selected sensible default
+need to run ML pipeline is one function: `run_ml()`. We've selected sensible default
```
Comment (zenalapp) [inline vignettes/introduction.Rmd:21]:
```suggestion
need to run the ML pipeline is one function: `run_ml()`. We've selected sensible default
```

File: vignettes/introduction.Rmd [hunk 2]
```diff
@@ -122,7 +122,7 @@ You may also want to provide:
 - A seed so that the results will be reproducible, and so that you get the same results as those you see here (i.e have the same train/test split).
 
 Say we want to use logistic regression, then the method we will use is `glmnet`.
-To do so, run ML with:
+To do so, run ML pipeline with:
```
Comment (zenalapp) [inline vignettes/introduction.Rmd:125]:
```suggestion
To do so, run the ML pipeline with:
```

**Review by zenalapp — APPROVED** [review]

**Comment by kelly-sovacool** [comment]
Note: I edited the issue description to fix the strikethroughs.

**Comment by kelly-sovacool** [comment]
Let's wait to merge this until @sklucas reviews so she can practice!

**Review by sklucas — APPROVED** [review]
Changes reviewed from all commits.

**Comment by kelly-sovacool** [comment]
@BTopcuoglu looks like @zenalapp made some suggestions that should be committed before you merge this.

**Comment by zenalapp** [comment]
Done.

**Pull request merged**